mslib/mscolab/file_manager.py

```diff
@@ -29,6 +29,7 @@
 import difflib
 import logging
 import git
+import threading
```

Comment:
please import only what we need from threading 
  
 from threading import Lock

Reply:
Why? I prefer having it namespace'd properly where I use it and it is not like `threading` is excessively long. Having it just be called `Lock` where it is used makes it ambiguous if it is coming from threading or from multiprocessing for example.

Reply:
good point, we keep it. 
We have no generel style for this, usually we use import when we use lots of its methods.

in both cases the module is always fully imported (no speedup by a selection, python parses the whole module).